examples/lb3-application/lb3app/common/models/coffee-shop.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const debug = require('debug')('loopback:example:lb3application');
+
+module.exports = function(CoffeeShop) {
+  CoffeeShop.status = function(cb) {
```

Comment:
Would you mind rewriting this function from callback-style to `async function`? We are not using callbacks in LB4 and async functions (functions returning a promise) have been supported by LB3 for long time.

```ts
 CoffeeShop.status = async function() {
  // ...
  return response;
}
```